tools/WindowsPorting/PerformDEPControlsPort.cmd

```diff
@@ -521,7 +521,8 @@ echo.
 echo Pulling latest from %1...
 echo.
 
-call git pull --force
+call git fetch
+call git merge FETCH_HEAD
```

Comment:
Is "FETCH_HEAD" a special git thing? Why not just git fetch at the top and then git checkout to the commit or origin ref? This process is never going to check anything in so it's better to be in detached head state all the time I would think.

Reply:
This is for OS ports, so we are going to be checking things into the porting branch - so being in detached head state isn't desirable in this case.  FETCH_HEAD refers to the commit ID at the head of the server-side branch, so this is equivalent to syncing to latest.